### PR TITLE
[macOS] Use variable instead of hard coded value

### PR DESCRIPTION
**Pull request by ddobranic**
# Description
Use the interval value in retry function
#### Related issue:
https://github.com/actions/runner-images/issues/7208
## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/helpers/Common.Helpers.psm1 [hunk 1]
```diff
@@ -130,8 +131,8 @@ function Start-DownloadWithRetry {
                 exit 1
             }
 
-            Write-Host "Waiting 30 seconds before retrying. Retries left: $Retries"
-            Start-Sleep -Seconds 30
+            Write-Host "Waiting '$Interval' seconds before retrying. Retries left: $Retries"
```
Comment (mikhailkoliada) [inline images/macos/helpers/Common.Helpers.psm1:134]: we can drop quotes on $Interval

Comment (ddobranic) [reply images/macos/helpers/Common.Helpers.psm1]: Updated.

**Review by sergei-pyshnoi — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**